### PR TITLE
updated team.ts for academic year 2022-23

### PR DESCRIPTION
**Pull request by NeerajHazarika**
# Related Issue
N/A

# Summary
updated team.ts for academic year 2022-23

# Proposed Changes
- added profile for Anirban Mukherjee, Rajesh Penugonda, Niket Bahety, Prince Kumar, Simeet Nayan, Divi Srivastava, Dhritin Dutta, Taaha Siddiqui Mohammed, Kiranmoy Saha, Jyoti Jaiswal.
- updated profile for Neeraj Pratap Hazarika, Arnab Sen, Sanjana Chakravarty, Sounak Majumder, Gaurav Sarkar, Sanandita Das, Rahul Halder, Anik Das, Vibali Joshi, Abhiroop Mukherjee, Sujal Modanwal, Anish De, Jyoti Anjaly, Krishnendu Bera, Rajdeep Ghosh, Abhinaba Chowdhury, Manish Kundu, Priyanshi Agarwal, Devanshu Chatterjee.
- fixed broken profile pic links (used linkedin profile if github profile pic doesnt exist)
- removed mentor role, and added all leads and team members of previous academic year to veteran role.
- added academic year along with role for junior to know CODE IIEST's history.

# Additional Info

# Checklist
- [x] Tested Changes
- [x] Translations
- [x] Documentation

# Screenshots

|           Original        | 
| -------------------- |
|  image |

|        Updated          |
| -------------------- |
|  image |

### CODE REVIEW

File: src/logic/team.ts [hunk 1]
```diff
@@ -156,42 +211,66 @@ export const members: Array<{name: string; role: string; avatar: string; twitter
...
     name: 'Devanshu Chatterjee',
-    role: 'Video Editor',
-    avatar: 'https://avatars.githubusercontent.com/u/78012571?s=400&u=b497b61c1dcdd0ed5b2b5c0f602bcef4698ee097&v=4',
+    role: 'Veteran, Video Editor 2021-22',
+    avatar: 'https://media.licdn.com/dms/image/C4D03AQGt5zAeGIX-wQ/profile-displayphoto-shrink_400_400/0/1653216699224?e=1677110400&v=beta&t=oIc10O7XA3XXsYQff9R3THYxssYJnyglKFRHkAEI0Jk',
     twitter: '',
     linkedin: 'https://www.linkedin.com/in/devanshu-chatterjee-517645200/',
     github: 'https://github.com/01-dc',
   },
+  {
```
Comment (sadn1ck) [inline src/logic/team.ts]: can move our seniors to alumni

Comment (NeerajHazarika) [reply src/logic/team.ts]: moved [👍](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwjRpJ7khoj8AhWyoekKHa09B4MQFnoECAoQAQ&url=https%3A%2F%2Femojipedia.org%2Fthumbs-up%2F&usg=AOvVaw3qbAeyzOrK0PiArZoxYZcj)

Comment (sadn1ck) [reply src/logic/team.ts]: i meant there's a separate alumni page (in alumni.ts) you can update them there according to year

**Comment by arnabsen1729** [comment]
https://github.com/codeiiest-dev/codeiiest-web/blob/v2/src/pages/team.vue#L63

Further, add a default avatar image. Eg. something like [this](https://www.dreamstime.com/default-avatar-profile-icon-vector-unknown-social-media-user-photo-default-avatar-profile-icon-vector-unknown-social-media-user-image184816085)

**Comment by NeerajHazarika** [comment]
pls review the changes made as requested

**Comment by sadn1ck** [comment]
<details>
<summary> netlify error log, pls fix </summary>
<br>
<pre>
8:11:52 AM: vite v2.3.8 building for production...
8:11:52 AM: transforming...
8:11:52 AM: ✓ 42 modules transformed.
8:11:52 AM: [vite-plugin-pwa] [vite]: Rollup failed to resolve import "public/default-avatar-icon.png" from "src/pages/team.vue?vue&type=script&setup=true&lang.ts".
8:11:52 AM: This is most likely unintended because it can break your application at runtime.
8:11:52 AM: If you do want to externalize this module explicitly add it to
8:11:52 AM: `build.rollupOptions.external`
8:11:52 AM: error during build:
8:11:52 AM: Error: [vite]: Rollup failed to resolve import "public/default-avatar-icon.png" from "src/pages/team.vue?vue&type=script&setup=true&lang.ts".
8:11:52 AM: This is most likely unintended because it can break your application at runtime.
8:11:52 AM: If you do want to externalize this module explicitly add it to
8:11:52 AM: `build.rollupOptions.external`
8:11:52 AM:     at onRollupWarning (/opt/build/repo/node_modules/.pnpm/vite@2.3.8/node_modules/vite/dist/node/chunks/dep-0ed4fbc0.js:45533:19)
8:11:52 AM:     at Object.onwarn (/opt/build/repo/node_modules/.pnpm/vite@2.3.8/node_modules/vite/dist/node/chunks/dep-0ed4fbc0.js:45323:13)
8:11:52 AM:     at Object.onwarn (/opt/build/repo/node_modules/.pnpm/rollup@2.51.2/node_modules/rollup/dist/shared/rollup.js:20250:20)
8:11:52 AM:     at ModuleLoader.handleResolveId (/opt/build/repo/node_modules/.pnpm/rollup@2.51.2/node_modules/rollup/dist/shared/rollup.js:19604:26)
8:11:52 AM:     at /opt/build/repo/node_modules/.pnpm/rollup@2.51.2/node_modules/rollup/dist/shared/rollup.js:19551:22
8:11:52 AM:     at processTicksAndRejections (internal/process/task_queues.js:95:5)
8:11:52 AM:     at async Promise.all (index 2)
8:11:52 AM:     at async ModuleLoader.fetchStaticDependencies (/opt/build/repo/node_modules/.pnpm/rollup@2.51.2/node_modules/rollup/dist/shared/rollup.js:19549:34)
8:11:52 AM:     at async Promise.all (index 0)
8:11:53 AM: Failed during stage 'building site': Build script returned non-zero exit code: 2 (https://ntl.fyi/exit-code-2)
8:11:52 AM:     at async ModuleLoader.fetchModule (/opt/build/repo/node_modules/.pnpm/rollup@2.51.2/node_modules/rollup/dist/shared/rollup.js:19525:9)
8:11:52 AM:  ERROR  Command failed with exit code 1. (https://ntl.fyi/exit-code-1)
8:11:52 AM: ​
8:11:52 AM:   "build.command" failed                                        
8:11:52 AM: ────────────────────────────────────────────────────────────────
8:11:52 AM: ​
8:11:52 AM:   Error message
8:11:52 AM:   Command failed with exit code 1: npx pnpm i --store=node_modules/.pnpm-store && npx pnpm run build (https://ntl.fyi/exit-code-1)
8:11:52 AM: ​
8:11:52 AM:   Error location
8:11:52 AM:   In build.command from netlify.toml:
8:11:52 AM:   npx pnpm i --store=node_modules/.pnpm-store && npx pnpm run build
8:11:52 AM: ​
8:11:52 AM:   Resolved config
8:11:52 AM:   build:
8:11:52 AM:     command: npx pnpm i --store=node_modules/.pnpm-store && npx pnpm run build
8:11:52 AM:     commandOrigin: config
8:11:52 AM:     environment:
8:11:52 AM:       - REVIEW_ID
8:11:52 AM:       - VITE_CALENDAR_KEY
8:11:52 AM:       - NPM_FLAGS
8:11:52 AM:       - NODE_VERSION
8:11:52 AM:     publish: /opt/build/repo/dist
8:11:52 AM:     publishOrigin: config
8:11:52 AM:   headers:
8:11:52 AM:     - for: /manifest.webmanifest
8:11:52 AM:       values:
8:11:52 AM:         Content-Type: application/manifest+json
8:11:52 AM:   headersOrigin: config
8:11:52 AM:   redirects:
8:11:53 AM:     - from: /*
      status: 200
      to: /index.html
  redirectsOrigin: config
Caching artifacts
</pre>
</details>

**Comment by arnabsen1729** [comment]
> netlify error log, pls fix

@NeerajHazarika Fix the issue with the avatar that I marked, that should ideally fix this.

**Comment by NeerajHazarika** [comment]
build passed now

image

**Comment by arnabsen1729** [comment]
LGTM

**Comment by arnabsen1729** [comment]
@NeerajHazarika do just resolve the conversations

**Comment by NeerajHazarika** [comment]
yep done

**Pull request merged**